packages/web/src/components/payment-method/PaymentMethod.tsx

```diff
@@ -130,16 +121,7 @@ export const PaymentMethod = ({
               popupZIndex={zIndex.USDC_ADD_FUNDS_FILTER_BUTTON_POPUP}
             />
           )
-        ) : null,
-      extraContent: shouldShowCoinflowAmountPicker ? (
```

Comment:
This effectively reverts Reed's changes anyway. Should we just revert that PR on main?

Reply:
I think it's better to keep the dead code in tbh, so it doesn't get very stale, but if you feel strongly, I'll revert as part of this PR. I'll let you make the call